src/Phinx/Db/Adapter/PdoAdapter.php

```diff
@@ -576,13 +576,20 @@ public function getAttribute($attribute)
      *
      * @param mixed $default Default value
      * @param string|null $columnType column type added
+     * @param int|null $precision datetime/timestamp column default value limit/precision
      *
      * @return string
      */
-    protected function getDefaultValueDefinition($default, $columnType = null)
+    protected function getDefaultValueDefinition($default, $columnType = null, $precision = null)
     {
-        if (is_string($default) && $default !== 'CURRENT_TIMESTAMP') {
-            $default = $this->getConnection()->quote($default);
+        if (is_string($default)) {
+            if (str_starts_with($default, 'CURRENT_TIMESTAMP')) {
```

Comment:
Should probably either use `strpos($default, 'CURRENT_TIMESTAMP') === 0` or add `symfony/polyfill-php80` to the composer.json dependencies as this function is coming from there and it is a sub-dependency of `symfony/console` (and probably others).